code/modules/vehicles/wheelchair.dm

```diff
@@ -0,0 +1,119 @@
+/obj/vehicle/ridden/wheelchair //ported from Hippiestation (by Jujumatic)
+	name = "wheelchair"
+	desc = "A chair with big wheels. It looks like you can move in this on your own."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair"
+	layer = OBJ_LAYER
+	max_integrity = 100
+	armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 30)	//Wheelchairs aren't super tough yo
+	legs_required = 0	//You'll probably be using this if you don't have legs
+	arms_requires = 0	//We'll be doing our own checks
```

Comment:
0 is the parent value for arms_requires on /obj/vehicle/ridden so it's unnecessary here, no?

also as far as I can tell, these two vars don't actually do anything on the parent obj either, they're just useless.

Reply:
it's for explicit